hclwrite/format_test.go

```diff
@@ -615,6 +615,16 @@ module "x" {
   abcde = "456"
 }`,
 		},
+		{
+			`attr = provider::framework::example()`,
```

Comment:
What happens if the input has spaces, are they being removed?

Reply:
Good question! Let me add a test case for that as well, which should provide an answer.